### ISSUE TITLE
Expose silentLoginCallback with useReactOidc hook

### ISSUE THREAD

**Issue by bmrfvr** [AxaGuilDEv/react-oidc#488]
### Issue and Steps to Reproduce
This package is incredibly helpful but for the flow that I need to use it for, I need to manually call silent login. Is it possible to expose that from the useReactOidc hook? 

Thanks! :)

**Comment by guillaume-chervet** [comment]
Hi @bmrfvr , thank you for your issue.

It seems to me a good idea.
@oyouf , what do you think?

**Comment by shyanta** [comment]
Hi @guillaume-chervet @oyouf ,

Is there any idea on when you guys maybe want to do this? For me as well it would be usefull to call this function manually.
In my case I am able to use the website either with or without a user session, only when my session on the id_server already exists, I want the user to also be logged in on my webapp.
On a previous version I worked with the vanilla version of oidc. What I did there was always trigger the silentLogin on first render to enable this functionality.

This would solve a lot for me as well, but if you have any other suggestions on how I can solve this, I'm all ears.

Thanks in advance! :+1:
